docs/getting-started/getting-started.md

```diff
@@ -0,0 +1,107 @@
+# Getting Started
+
+## Before you begin
+
+### Software requirements
+- Eclipse IDE. Minimum Eclipse IDE version: 2222-03.
+- Java runtime. Minimum Java version: 11.
+- [Open Liberty tools for Eclipse plugin](../installation/installation.md).
+
+### Application requirements
+
+- Define a Liberty server.xml configuration file at location src/main/liberty/config.
+- Configure the [Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven#configuration) or [Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle#adding-the-plugin-to-the-build-script)
+
+## Opening the Liberty tools dashboard view
+
+Click on the Open Liberty icon on the toolbar.
+
+![Dashboard context menu for Maven Project](images/openLibertyIconOnToolbar.png)
+
+Alternatively, click on the `Open Dashboard` option in the `Liberty` menu. 
+
+![Dashboard context menu for Maven Project](images/openLibertyToolMenuDashboardView.png)
+
+If the dashboard is opened and there application projects that already properly configured to run on Liberty and use Liberty dev mode, those projects are automatically added to the dashboard when it opens.
```

Comment:
If the dashboard is opened and there are application projects that are already properly configured to run on Liberty and use Liberty dev mode,